packages/config/src/lib/addPrefixToContent.ts

```diff
@@ -0,0 +1,40 @@
+//TODO - iterate through a string and save indexes of the curly parentheses [openingIndex, closingIndex]
+// slice the selector before that, if it's top level selector, add prefix in a scope above, if its nested, dont care
+
+const addPrefixToContent = (content: string, sassPrefix: string): string => {
+  // Stack to store pairs of opening and closing curly brace indexes
```

Comment:
Maybe instead of building a custom parser, we can utilise sass' `compileString` function? https://sass-lang.com/documentation/js-api/functions/compilestring/

Reply:
Hm. I thought maybe compileString would allow to have a more structured representation of the css, but it's just a string too.

Reply:
Oh, we don't want to compile, we will leave that to saas. We just want to not prefix css rules that match the app name. To allow styling the root element. I think is a manual process. I wish there was some nice AST for scss that is not 10 years old.